scripts/dumps/bulk_genome_dump_from_core.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper of dump_genome_from_core.pl to dump a list of FASTA files
+"""
+import argparse
+import subprocess
+import os
+import sys
+import logging
+import yaml
+
+
+def setup_logging():
+    """
+    Sets up logging configuration.
+    """
+    logging.basicConfig(
+        level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+    )
```

Comment:
Is it necessary to do the basic logging config here, or could it be done in the `main` function?

Reply:
I prefer adding this functionality into a functional for good reading

Reply:
Thanks for pointing it out. I removed the `logging.basicConfig` directly in the `main` function